lib/utils/presentation.js

```diff
@@ -85,7 +87,46 @@ var generateWhitelist = {
 	}
 }
 
+/**
+ * Return the sessionId if the given slideshow is actived by given user, otherwise return null
+ * 
+ * @method isLiveBy
+ * @param ObjectId slideshowId   the slideshow's Id
+ * @param ObjectId userId 
+ * @return ObjectId return the sessionId if the given slideshow is actived by given user, otherwise return null
+ */
+var isLiveBy = coroutine(function* isLiveGen(userId, slideshowId) {
```

Comment:
`isLiveBy` should return true or false. `getSessionIfLiveBy` is more descriptive for what you are trying to do.